user_docs/en/changes.t2t

```diff
@@ -43,6 +43,7 @@ Please refer to [the developer guide https://www.nvaccess.org/files/nvda/documen
 - Note: this is an Add-on API compatibility breaking release.
 Add-ons will need to be re-tested and have their manifest updated.
 - Added extension point: ``treeInterceptorHandler.post_browseModeStateChange``. (#14969, @nvdaes)
+- It is now possible to use plural forms in addon's translations. (#15661, @beqabeqa473)

```

Comment:
If this currently works in 2023.2 and 2023.3, then fixing this is an unreleased regression and doesn't need a change log entry

```suggestion
```

Reply:
Thanks @beqabeqa473 - Fix grammar

```suggestion
- It is now possible to use plural forms in an add-on's translations. (#15661, @beqabeqa473)
```